mainnet/xdc-attach.sh

```
@@ -1 +1 @@
-sudo docker exec -it mainnet_xinfinnetwork_1 XDC attach /work/xdcchain/XDC.ipc
\ No newline at end of file
+sudo docker exec -it xdcnetwork-mainnnet-node XDC attach /work/xdcchain/XDC.ipc
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)